mtp_noms_ops/templates/security/checks_list.html

```diff
@@ -3,112 +3,128 @@
 {% load mtp_common %}
 {% load security %}
 
-{% block page_title %}{{ view.title }} – {{ block.super }}{% endblock %}
-
+{% block page_title %}
+    {{ view.title }}
+    –
+    {{ block.super }}
+{% endblock %}
 
 {% block inner_content %}
-  <header>
+
+<header>
     <h1 class="heading-xlarge">{{ view.title }}</h1>
-  </header>
+</header>
+<aside class="mtp-object-list-toggle print-hidden">
+    <a href="/security/credits-history/" class="button-secondary">View decision history</a>
+</aside>
 
-  {% include 'mtp_common/includes/message_box.html' %}
+{% include 'mtp_common/includes/message_box.html' %}
 
-  <p class="font-large">
+<p class="font-large">
     {% blocktrans trimmed count count=form.total_count with number=form.total_count|separate_thousands %}
-      {{ number }} credit
+    {{ number }}
```

Comment:
please don't wrap translated strings. it won't work properly for the translation templates

Reply:
same goes for the few places below